AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -2259,15 +2310,7 @@ - (CGSize)calculatedSize
 - (ASSizeRange)constrainedSizeForCalculatedLayout
 {
   ASDN::MutexLocker l(__instanceLock__);
-  return _calculatedLayout.constrainedSizeRange;
-}
-
-- (void)setLayoutSpecBlock:(ASLayoutSpecBlock)layoutSpecBlock
```

Comment:
Are we dropping support for layoutSpecBlock:? If not, looks like there needs to be a locker in this method…